hr_infraction/README.rst

```diff
@@ -0,0 +1,144 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+HR Infraction Management
+========================
+
+This module add employee infraction management
+
```

Comment:
Can you explain a bit more the concept of "infraction management" and the scope of the module here? An "executive summary" would be nice to help people decide if the module applies to their need.